mlflow/keras/callback.py

```diff
@@ -85,19 +83,20 @@ def on_epoch_end(self, epoch, logs=None):
         """Log metrics at the end of each epoch."""
         if not self.log_every_epoch or logs is None:
             return
-        self.metrics_logger.record_metrics(logs, epoch)
+        log_metrics(logs, step=epoch, synchronous=False)
```

Comment:
curious why we need this change

Reply:
good question - a few months ago we added async logging support to our fluent API, so we on longer need to use the specific logger as before.